core/sup_tree_core/gear_config_poller.ex

```diff
@@ -20,6 +22,8 @@ defmodule AntikytheraCore.GearConfigPoller do
 
   @interval 60_000
 
+  @typep state :: %{last_checked_at: integer}
```

Comment:
Declaring this as `@typep` is :+1:
We can make the value type more specific by `integer` => `pos_integer`, as seconds since UNIX epoch is always positive.

Reply:
fixed in ba40bc0